assets/vsphere_features_config.yaml

```diff
@@ -17,6 +17,7 @@ data:
   "improved-volume-topology": "false" # operator will set this value to true if cluster supports CSI topology
   "improved-csi-idempotency": "true" # Causes operator to create CRs for volume operations for sync them with k8s server
   "multi-vcenter-csi-topology": "true" # add support multiple vcenters. Although we don't support this feature, having it disabled causes driver to crash loop.
+  "max-pvscsi-targets-per-vm": "true" # enables higher limit (255) in the driver for volumes per node in vSphere 8, the limit value is set by `MAX_VOLUMES_PER_NODE`
```

Comment:
This is already enabled, so setting this doesn't do anything fwiw.

Reply:
I see now, thank you! Explained here: https://github.com/openshift/enhancements/pull/1748#discussion_r1943511577